### PR TITLE
corrected some minor mistakes on position, display and floats

### PR DESCRIPTION
**Pull request by francodelera**
As we talked on the daily meeting, here are the corrections to the sign up form and the css on which i had some mistakes specially with position, display and floats.

### CODE REVIEW

File: 00-HTML-CSS-basics/Excercise1/signupForm.html [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
+        <label for="birthday">Birthday </label><input type="date" id="birthday"><br>
+        <label for="sports">Select your favorite sport </label><select id="sports">
             <option value="soccer">Soccer</option>
             <option value="basketball">Basketball</option>
             <option value="tennis">Tennis</option>
             <option value="other">Other</option>
         </select>
         <br><label for="textarea">Tell us about yourself</label>
-        <br><textarea rows="6" cols="35">My hobbies are...</textarea>
+        <br><textarea  id="textarea" rows="6" cols="35">My hobbies are...</textarea>
```
Comment (ruggero-nardi) [inline 00-HTML-CSS-basics/Excercise1/signupForm.html]: This is fine but it's recommended to use ids that doesn't match the tag name, something like id="hobbiesTextarea"

**Review by ruggero-nardi — CHANGES_REQUESTED** [review]
Everything seems fine, only one thing to correct

File: 00-HTML-CSS-basics/Excercise1/signupForm.html (hunk 1 above)
Comment (francodelera) [reply 00-HTML-CSS-basics/Excercise1/signupForm.html]: Modified the file according to the recommendations.

Comment (francodelera) [reply 00-HTML-CSS-basics/Excercise1/signupForm.html]: New changes made to the layouts excercise.

**Review by ruggero-nardi — APPROVED** [review]

**Comment by ruggero-nardi** [comment]
Good job with the new ids and the use of flexbox

**Pull request merged**